.github/workflows/pull-request.yml

```
@@ -94,11 +89,11 @@ jobs:
       - run: npm ci
 
       - name: Test
-        if: ${{ inputs.cron-run == 'false' || inputs.cron-run == '' }}
+        if: ${{ github.event_name == 'pull_request' }}
         run: npm run affected:test -- --browsers=ChromeHeadless --watch=false $TEST_OPTS --base=origin/develop
 
       - name: Test all
-        if: ${{ inputs.cron-run == 'true' }}
+        if: ${{ github.event_name == 'schedule' }}
```

```suggestion
        if: ${{ github.event_name != 'pull_request' }}
```

should be more future-proof

---

Done